# Remove dependencies on user32.winxp.lib and user32-delay.dll in GYP
# user32.winxp.lib is deleted in source cleaning (which causes building to fail) and Google no longer supports Windows XP

--- a/chrome/chrome_dll.gypi
+++ b/chrome/chrome_dll.gypi
@@ -198,23 +198,23 @@
                       # inadvertently taking dependencies on post-XP user32
                       # exports.
                       'AdditionalDependencies!': [
-                        'user32.lib',
+                        #'user32.lib',
                       ],
                       'IgnoreDefaultLibraryNames': [
-                        'user32.lib',
+                        #'user32.lib',
                       ],
                       # Remove user32 delay load for chrome.dll.
                       'DelayLoadDLLs!': [
-                        'user32.dll',
+                        #'user32.dll',
                       ],
                       'AdditionalDependencies': [
-                        'user32.winxp.lib',
+                        #'user32.winxp.lib',
                       ],
                       'DelayLoadDLLs': [
-                        'user32-delay.dll',
+                        #'user32-delay.dll',
                       ],
                       'AdditionalLibraryDirectories': [
-                        '<(DEPTH)/build/win/importlibs/x86',
+                        #'<(DEPTH)/build/win/importlibs/x86',
                       ],
                       'ForceSymbolReferences': [
                         # Force the inclusion of the delay load hook in this
